README.md

```diff
@@ -19,3 +19,23 @@ Use the `ppid` parser function:
 ```
 {{#ppid:}}
 ```
+
+## PHP Configuration
```

Comment:
Idea is to have it on the page, similar to https://professional.wiki/en/extension/page-approvals#Configuration. So we would need to move it later

Reply:
I'll leave all the README updates for https://github.com/ProfessionalWiki/PersistentPageIdentifiers/issues/35